nasa-gibs/worldview
# Add 2 PACE layers

## Description

Fixes #WV-3152 .

Added two PACE/OCI layers - Corrected Reflectance (True Color) and Chlorophyll a

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-189.7068165124282,-17.49447340178364,-24.971272940875778,55.30576417445151&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,OrbitTracks_PACE_Ascending,OCI_PACE_Chlorophyll_a,OCI_PACE_True_Color,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-04-23-T14%3A03%3A54Z)
2. Look for the PACE/OCI layers and make sure they appear on the map, appear in the correct measurements (Corrected Reflectance and Chlorophyll a) and that you can take image snapshots of them.


@nasa-gibs/worldview
